roles/wisdom_warmup_model/templates/wisdom-warmup_entrypoint.yml.j2

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: wisdom-warmup-entrypoint
+  namespace: {{ wisdom_namespace }}
+data:
+  entrypoint.sh: |-
+    #! /bin/bash
+    CONCURRENCY={{ warmup_concurrency }}
+    TOTAL_REQUESTS={{ warmup_total_requests }}
```

Comment:
same comment as the other role, I would move this to `files/entrypoint.sh` and define `CONCURRENCY` and `TOTAL_REQUESTS` as part of the Pod `env`

Reply:
and `WISDOM_NAMESPACE`